### ISSUE TITLE
Parallel running issue-PUFoam

### ISSUE THREAD

**Issue by sahrishNaqvi** [karimimp/PUFoam#1]
Hi Karimi, this is Sahrish Batool Naqvi, I am working with your open-source code PUFoam and have facing some issues regarding parallel running. Does it have the capability to run parallel? Please shed a light in to this issue.

**Comment by karimimp** [comment]
Hi,
That's pretty good test case to try. Have you also tried a simple cup test case to run it in parallel. I would say to look at the moments and check if the gelling point is reached in one or two cells in the domain
Can you please email me (mohsenk@outlook.com) the pdf file, I do not see any attached doc here

**Comment by sahrishNaqvi** [comment]
Yes, I tried simple cup case (provide by you on GitHub together with the code) and parallel running is working perfectly fine in that case.  

Let me have a deep look into the moments and to follow the gelling point, and actually this is what not happening, that's why solver got diverge. Thanks for this suggestion, it will save my time in order to track the hidden needle. 

I have attached that document again and included your alternative email. Please let me know if you find it, else I will copy paste it here. 

BR

Sahrish

    On Wednesday, 30 March 2022, 14:43:01 WEST, Mohsen ***@***.***> wrote:  
 
 


Hi,
That's pretty good test case to try. Have you also tried a simple cup test case to run it in parallel. I would say to look at the moments and check if the gelling point is reached in one or two cells in the domain
Can you please email me ***@***.***) the pdf file, I do not see any attached doc here

**Comment by sahrishNaqvi** [comment]
Thank you so much. Sure I will keep disturbing you.
Sahrish
    On Tuesday, 29 March 2022, 18:09:02 BST, Mohsen ***@***.***> wrote:  
 
 


Sounds like a really good plan and please do not hesitate to write to me, I'd be glad to help.
